contributions/catalog/webapps/static/css/style.css

```diff
@@ -177,83 +136,80 @@ input[type=reset]:hover {
 }
 
 .button {
-    background-color: #4CAF50; /* Green */
     border: none;
     border-radius: 4px;
     color: white;
-    padding: 10px 10px;
+    padding: 5px 10px 5px 10px;
     text-align: center;
     display: inline-block;
-    font-size: 15px;
+    font-size: 14px;
     margin: 5px 5px;
     transition-duration: 0.2s;
     cursor: pointer;
     float: none;
+    height: 36px;
+    line-height: 20px;
 }
 
 .button1 {
-    background-color: white;
-    color: black;
-    border: 1px solid #4CAF50;
     display: inline-block;
-}
-
-.button1:hover {
-    background-color: #4CAF50;
-    color: white;
-    display: inline-block;
-}
-
-.button2 {
-    background-color: white;
     color: black;
-    border: 1px solid #008CBA;
-    display: inline-block;
+    border-radius: 24px;
+    border: solid 2px #002855;
+    background-color: #f5f5f5;
 }
 
-.button2:hover {
-    background-color: #008CBA;
+.button1:hover {
+    background-color: #002855;
     color: white;
     display: inline-block;
 }
 
-.button3 {
-    color: black;
+.button2, .button2:hover {
+    color: #002855;
     display: inline-block;
     border-radius: 24px;
     border: solid 2px #e84a27;
     background-color: #fff;
+    text-decoration: none;
+    font-weight: 500;
```

Comment:
```suggestion
    font-weight: bold;
```
Font-weight is marked as bold in Zeplin. Please fix this in this and other buttons too. 
![image](https://user-images.githubusercontent.com/4306604/214933338-c78e9467-eb1f-4b02-ac5d-631c4af794ac.png)

Reply:
updated font weight for all buttons.